dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentsExportControllerTest.java

```diff
@@ -203,6 +204,94 @@ void shouldGetEnrollmentsWithAttributes(
     assertHasMember(attribute, "code");
   }
 
+  @Test
+  void shouldGetEnrollmentWithoutRelationshipsWhenRelationshipIsDeletedAndIncludeDeletedIsFalse() {
+    Relationship relationship = get(Relationship.class, "p53a6314631");
+    manager.delete(relationship);
+
+    assertNotNull(
+        relationship.getTo().getEnrollment(),
+        "test expects relationship to have a 'to' enrollment");
+    Enrollment enrollment = relationship.getTo().getEnrollment();
+
+    JsonList<JsonRelationship> jsonRelationships =
+        GET(
+                "/tracker/enrollments?enrollments={id}&fields=*&includeDeleted=false",
```

Comment:
Our tests should probably also only ask for what they need in the `fields` to make them as slow as necessary 😅